tests20/python_client/testcases/test_partition.py

```diff
@@ -41,15 +44,13 @@ def test_partition_empty_name(self, partition_name):
         method: 1. create a partition empty none name
         expected: 1. raise exception
         """
-        self.collection_wrap.init_collection()
-
         # create a collection
         collection_w = self.init_collection_wrap()
 
         # create partition
         self.partition_wrap.init_partition(collection_w.collection, partition_name,
                                            check_task=CheckTasks.err_res,
-                                           err_code=1, err_msg="Partition tag should not be empty")
+                                           check_items={"err_code": 1, "err_msg": "Partition tag should not be empty"})
```

Comment:
I am afraid of spelling mistakes. how about this
```
err_code = "err_code"
err_msg = "err_msg"
check_items={err_code: 1, err_msg: "is illegal"}
```

Reply:
good idea. please add 2 public variables in your commits :)